wiki/tips.md

```diff
@@ -0,0 +1,91 @@
+---
+author: hifish
+desc: This page has small general tips and tricks.
+lastUpdated: 2025-01-11T23:44:26.934Z
+title: Tips & Tricks
+---
+# Tips & Tricks
+## <h2 id="inserting" sidebar="Sprite Layering">Add sprite/object behind or above another</h2>
+
+You may have a sprite that you might wanna put above or below another sprite. This is possible using the `insert(position:Int, object:T)` function.
+```haxe
+var blueSolid = new FunkinSprite(200, 200).makeSolid(200, 200, FlxColor.RED);
+add(blueSolid);
+
+var redSolid = new FunkinSprite(250, 250).makeSolid(100, 100, FlxColor.RED);
+insert(members.indexOf(blueSolid)+1, redSolid);
+/*
+This would add the red solid right above the blue solid. If we want it below the blue
+solid, we would use the insert function, like so.
+*/
+insert(members.indexOf(blueSolid), redSolid);
```

Comment:
can you confirm that this is how it works
and mention how using the -1 or +1 is incorrect if you want to do x